### DISCUSSION TITLE
Adding recurring flights

### DISCUSSION THREAD

**Discussion by anantone** [johanohly/AirTrail#22] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Thank you for this project, quite exciting!

Was wondering if it would be possible to add recurring flights, for instance if I want to track the same flight every day (say, Paris-New York on United at 14h10, 7 days a week)? This way, wouldn't have to add it manually each time a family member takes a flight to visit (they always take the same company, at the same time...)

**Comment by palitu** [comment]
Interesting concept.

In Australia we often have Fly-in Fly-out workers, who take the same flights every week or second week, depending on their roster.  something like this could work for them.

**Comment by johanohly** [comment]
It is definitely an interesting idea, but I am afraid that not enough people want this to justify developing it. 
I will let this discussion stay open to see if enough interest develops.
